docs/modules/ROOT/pages/documentation/payara-server/jbatch/asadmin.adoc

```diff
@@ -116,11 +116,42 @@ the classes mentioned:
 </dependency>
 ----
 
-_Since Payara Server 5.183_
+[[clean-jbatch-repository]]
+== `clean-jbatch-repository`
+
+_Since Payara Server 5.25.0_
+
+*Usage*::
+`asadmin> clean-jbatch-repository --status COMPLETED|ALL --days <int.value> <JobName>
+
+*Aim*::
+The clean-jbatch-repository subcommand deletes records of job executions
+
+[[command-options-clean]]
+=== Command Options
+
+[cols=",,,,",options="header",]
+|=======================================================================
+|Option | Type |Description |Default |Mandatory
+|`--status` | ALL/COMPLETED| The status of batch jobs executions to which delete the records of.
```

Comment:
For better readability, use a bullet list to list the types:
* ALL
* COMPLETED